packages/manager-components/src/components/region/region.component.tsx

```diff
@@ -0,0 +1,21 @@
+import { useTranslation } from 'react-i18next';
+import './translations/region';
+import './translations/datacenter';
+
+export interface RegionProps {
+  mode: 'region' | 'datacenter';
+  name: string;
+  micro?: number;
+}
+
+export function Region({ mode = 'region', name, micro }: RegionProps) {
+  const { t } = useTranslation(mode === 'region' ? 'region' : 'datacenter');
+
+  return (
+    <>
+      {mode === 'region' ? t(`region_${name}`) : t(`region_${name}`, { micro })}
```

Comment:
```suggestion
      {t(`region_${name}`, { micro })}
```

passing `micro: undefined` won't affect the translation in the case of region